app/src/examples/AnimatedSensorMagneticFieldExample.tsx

```diff
@@ -0,0 +1,39 @@
+import Animated, {
+  SensorType,
+  useAnimatedSensor,
+  useAnimatedStyle,
+} from 'react-native-reanimated';
+import { StyleSheet, View } from 'react-native';
+
+import React from 'react';
+
+export default function AnimatedSensorMagneticFieldExample() {
+  const gravity = useAnimatedSensor(SensorType.MAGNETIC_FIELD);
+
+  const animatedStyle = useAnimatedStyle(() => {
+    const { x, y } = gravity.sensor.value;
+    const angle = (Math.atan2(y, x) * 180) / Math.PI;
+    return {
+      transform: [{ rotateZ: `${angle}deg` }, { translateY: 75 }],
```

Comment:
Why do we need `{ translateY: 75 }` here?

Would it be possible to use a different prop (e.g. `marginTop: -75`) and move this style to `StyleSheet.create`?

Alternatively, we could define `const BOX_SIZE = 150;` and use `{ translateY: BOX_SIZE / 2 }` here.